lib/mayaUsd/fileio/primUpdater.h

```diff
@@ -49,71 +51,56 @@ class UsdMayaPrimUpdater
         Push = 1 << 0,
         Pull = 1 << 1,
         Clear = 1 << 2,
+        AutoPull = 1 << 3,
         All = Push | Pull | Clear
```

Comment:
IDK if AutoPull should be in All?

Reply:
I don't know either.  I will assume the above is correct, until shown otherwise.